code/src/java/gmgen/plugin/dice/Die.java

```diff
@@ -20,45 +20,34 @@
  *
  *  Created on January 24, 2002, 11:15 AM
  */
-package gmgen.plugin;
+package gmgen.plugin.dice;
 
 import java.util.Random;
 
-/** Abstract class describing a die of any kind
- * @author Soulcatcher
- * @since May 24, 2003
```

Comment:
This particular file has been basically stripped to the bare minimum and no longer directly reflects the work of the author. I can restore of required, but its of questionable value (vcs stores this history)